src/request/ask.rs

```diff
@@ -35,16 +35,17 @@ where
 {
     mailbox: &'a Mb,
     signal: Signal<A>,
-    rx: oneshot::Receiver<Result<BoxReply, BoxSendError>>,
+    rx: oneshot::Receiver<Result<BoxReply, error::BoxSendError>>,
 }
 
 /// A request to a remote actor.
 #[allow(missing_debug_implementations)]
+#[cfg(feature = "remote")]
 pub struct RemoteAskRequest<'a, A, M>
```

Comment:
Seems like this type doesn't get marked as feature gated when documenting due to the missing `#[cfg_attr(feature = "doc", doc(cfg(feature = "remote")))]` annotation.

Reply:
Edit: if we go with the `doc_auto_cfg` suggestion, then we won't need this anymore